src/activity_handlers.cpp

```diff
@@ -2023,6 +2024,30 @@ void activity_handlers::vehicle_finish( player_activity *act, player *p )
     }
 }
 
+void activity_handlers::hand_crank_do_turn( player_activity *act, player *p )
+{
+    // Hand-crank chargers seem to range from 2 watt (very common easily verified)
+    // to 10 watt (suspicious claims from some manufacturers) sustained output.
+    // It takes 2.4 minutes to produce 1kj at just slightly under 7 watts (25 kj per hour)
+    // time-based instead of speed based because it's a sustained activity
+    act->moves_left -= 100;
+    item &hand_crank_item = p ->i_at( act->position );
+
+    // TODO: This should be 144 seconds, rather than 24 (6-second) turns
+    // but we don't have a seconds time macro?
+    if( calendar::once_every( 24_turns ) ) {
```

Comment:
You can change this to 
```suggestion
    if( calendar::once_every( 144_seconds ) ) {
```

Reply:
I swear I tried that and got an error. Let me try again.

Reply:
```
src/activity_handlers.cpp:2038:34: error: no matching literal operator for call to 'operator""_seconds' with argument of type 'unsigned long long' or 'const char *', and no matching literal operator template
    if( calendar::once_every( 144_seconds ) ) {
                                 ^
```

Both VS Code and clang say this is not legal.

Reply:
That problem is the origin of the comment immediately above the line, btw.

Reply:
Well, we could circumvent that by using something like `2_minutes`.